make/migrations/postgresql/0030_1.11.0_schema.up.sql

```diff
@@ -0,0 +1,26 @@
+/* TODO remove the table artifact_2 and use the artifact instead after finishing the upgrade work */
+CREATE TABLE artifact_2
```

Comment:
We need to decide the uuid .vs. numeric id stuff...

Reply:
There is already an artifact table introduced by quota, it's numeric ID, the `artifact_2` is a temp one, so it's ID is numeric. And for the consistence, I think the numeric ID for table `tag` is better

Reply:
we should consider the migration issue. 
1 from non-artifact scheme to artifact_2
2 from artifact to artifact_2.

Reply:
What's the meaning of `from non-artifact scheme to artifact_2`?

Reply:
@wy65701436 Will cover this in the upgrade&migration part

Reply:
@reasonerjt We can update the type of ID when we decide to use uuid or numeric ID, this is a temp table